modules/genomics_cromwell/cromdb.tf

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "random_password" "cromwell_db_pass" {
+  length  = 16
+  special = false
+}
+module "cromwell-mysql-db" {
```

Comment:
Terraform resource names should be split by underscores.  Can you validate all the other resources as well?  This should be `cromwell_mysql_db`.